projects/angular/src/popover/dropdown/_menu-mixins.clarity.scss

```
@@ -16,110 +17,47 @@
-    variables.$clr-global-borderwidth,
-    variables.$clr-use-custom-properties
-  );
+  background-color: dropdown-variables.$clr-dropdown-bg-color;
+
+  border-width: variables.$clr-global-borderwidth;
   border-style: solid;
-  @include mixins.css-var(
-    border-color,
-    clr-dropdown-border-color,
-    dropdown-variables.$clr-dropdown-border-color,
-    variables.$clr-use-custom-properties
-  );
-  @include mixins.css-var(
-    border-radius,
-    clr-dropdown-border-radius,
-    variables.$clr-global-borderradius,
-    variables.$clr-use-custom-properties
-  );
-  // OldIE/Edge
-  box-shadow: 0 variables.$clr-global-borderwidth variables.$clr_baselineRem_0_125
-    dropdown-variables.$clr-dropdown-box-shadow;
-  @if variables.$clr-use-custom-properties == true {
-    box-shadow: 0
-      variables.$clr-global-borderwidth
-      variables.$clr_baselineRem_0_125
-      var(--clr-dropdown-box-shadow, dropdown-variables.$clr-dropdown-box-shadow);
-  }
-  margin-top: variables.$clr_baselineRem_2px;
-  @include mixins.css-var(
-    padding,
-    clr-dropdown-items-padding,
-    variables.$clr_baselineRem_0_5 0,
-    variables.$clr-use-custom-properties
-  );
+  border-color: dropdown-variables.$clr-dropdown-border-color;
+  border-radius: dropdown-variables.$clr-dropdown-border-radius;
+
+  box-shadow: 0 variables.$clr-global-borderwidth mixins.baselinePx(3) dropdown-variables.$clr-dropdown-box-shadow;
+
+  margin-top: tokens.$cds-global-space-2;
+  padding: dropdown-variables.$clr-dropdown-items-padding;
   visibility: hidden;
   z-index: map.get(variables.$clr-layers, dropdown-menu);
 }
 
 @mixin generate-dropdown-item() {
-  @include mixins.css-var(
-    color,
-    clr-dropdown-item-color,
-    dropdown-variables.$clr-dropdown-item-color,
-    variables.$clr-use-custom-properties
-  );
-  font-size: dropdown-variables.$clr-dropdown-item-font-size;
-  @include mixins.css-var(
-    font-weight,
-    clr-dropdown-item-font-weight,
-    dropdown-variables.$clr-dropdown-item-font-weight,
-    variables.$clr-use-custom-properties
-  );
-  letter-spacing: dropdown-variables.$clr-dropdown-item-letter-spacing;
+  @include mixins.generate-typography-token('SECONDARY-13-RG-EXP');
 
+  color: dropdown-variables.$clr-dropdown-item-color;
   background: transparent;
   border: 0;
   cursor: pointer;
   display: block;
   height: auto;
-  line-height: inherit;
   margin: 0;
   width: 100%;
   text-transform: none;
 
   &:hover {
-    @include mixins.css-var(
-      background-color,
-      clr-dropdown-bg-hover-color,
-      dropdown-variables.$clr-dropdown-bg-hover-color,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      color,
-      clr-dropdown-item-color,
-      dropdown-variables.$clr-dropdown-item-color,
-      variables.$clr-use-custom-properties
-    );
+    background-color: dropdown-variables.$clr-dropdown-bg-hover-color;
+    color: dropdown-variables.$clr-dropdown-item-color;
```

Should use different internal token based on CDS spec.

```suggestion
    color: tokens.$cds-alias-object-interaction-color-hover;
```

---

done